test/info.py

```diff
@@ -26,42 +26,90 @@
     'sam3u2c': 0x97969903,
 }
 
+<<<<<<< HEAD
```

Comment:
This looks like an unresolved merge conflict. I'm guessing this wasn't intentional.